config.go

```diff
@@ -22,6 +24,7 @@ type Config struct {
 		Token         string
 		IconURL       string
 		SkipTLSVerify bool
+		IrcNickPrefix string
```

Comment:
Same here, for compatibility, could you make the default "irc-" if IrcNickPrefix is ""